swaywm/wlroots
# xwm clipboard sync

Sync the Wayland clipboard selection with the xwm clipboard.

Tests:

* Copy from a Wayland window and paste into an Xwayland window.
* Copy from an Xwayland window and paste into a Wayland window.
* <s>`sleep 5 && xclip -o`, change focus to a wayland window, and modify the clipboard. xclip should not receive an updated value.</s> not in scope
